src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callBack = compareFunction;
```

Comment:
You can directly assign `compareFunction` to `callBack` without checking if it's not null. If `compareFunction` is null, `callBack` will also be null which is what we want.